### PR TITLE
Fix the flaky test in the two-choice random cache eviction policy

### PR DESCRIPTION
**Pull request by ChunxuTang**
### What changes are proposed in this pull request?

Fix the flaky test in the two-choice random cache eviction policy

### Why are the changes needed?

As the test is flaky, it will influence other PRs.

### Does this PR introduce any user facing changes?

No.

### CODE REVIEW

File: core/client/fs/src/test/java/alluxio/client/file/cache/TwoChoiceRandomEvictorTest.java [hunk 1]
```diff
@@ -39,8 +39,6 @@ public void before() {
   public void evictGetOrder() {
     mEvictor.updateOnGet(mFirst);
     Assert.assertEquals(mFirst, mEvictor.evict());
-    mEvictor.updateOnGet(mSecond);
-    Assert.assertEquals(mSecond, mEvictor.evict());
```
Comment (lucyge2022) [inline core/client/fs/src/test/java/alluxio/client/file/cache/TwoChoiceRandomEvictorTest.java:43]: or u could do Assert.assertTrue(evictedPage.equals(mFirst) || evictedPage.equals(mSecond)); as well here?

**Review by lucyge2022 — APPROVED** [review]
LGTM thanks

File: core/client/fs/src/test/java/alluxio/client/file/cache/TwoChoiceRandomEvictorTest.java (hunk 1 above)
Comment (ChunxuTang) [reply core/client/fs/src/test/java/alluxio/client/file/cache/TwoChoiceRandomEvictorTest.java:43]: Yep! Thanks for your suggestion. I updated the test here.

**Review by beinan — APPROVED** [review]
lgtm

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**